openfisca_aotearoa/tests/social_security/child_disability_allowance.yaml

```diff
@@ -31,7 +31,7 @@
       - false  # Tama (disabled child)
       - false  # tamahine (able bodied)
 
-- name: Tests - Eligible
+- name: Child Disability Allowance, eligible
```

Comment:
@Br3nda I think it would be nice if we had a naming convention, for nice display